source/Jobbr.Execution.Forked.nuspec

```diff
@@ -13,14 +13,16 @@
     <icon>images\icon.png</icon>
     <dependencies>
       <group targetFramework="net6.0">
-        <dependency id="Jobbr.ComponentModel.Registration" version="[1,1.1.2)"/>
-        <dependency id="Jobbr.ComponentModel.Execution" version="[1,1.1.2)"/>
+        <dependency id="Jobbr.ComponentModel.Registration" version="[2.9,3.1)"/>
+        <dependency id="Jobbr.ComponentModel.Execution" version="[2.9,3.1)"/>
       </group>
     </dependencies>
```

Comment:
Can you add an empty group with `<group targetFramework="netstandard2.0" />` so to remove the nuget pack warning message:

> Attempting to build package from 'Jobbr.Execution.Forked.nuspec'.
WARNING: NU5128: Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the other location. Consult the list of actions below:
> - Add a dependency group for .NETStandard2.0 to the nuspec
Successfully created package 'C:\projects\jobbr-execution-forked\Jobbr.Execution.Forked.3.1.0-include-runtime-dll.1.nupkg'.
nuget pack source\Jobbr.Runtime.ForkedExecution.nuspec -version "%GitVersion_SemVer%" -prop "target=%CONFIGURATION%"